target/linux/ramips/dts/mt7620a_asus_rt-ac51u.dts

```diff
@@ -60,7 +60,7 @@
 
 		enable-leds {
 			gpio-export,name = "enable-leds";
-			gpio-export,output = <1>;
+			gpio-export,output = <0>;
 			gpios = <&gpio0 10 GPIO_ACTIVE_HIGH>;
```

Comment:
gpio_export has not been accepted upstream, so it is kind of deprecated.

If you are touching it anyway, consider changing it to a gpio-hog, should be something like:

```
+&gpio0 {
+    leds-enable {
+        gpio-hog;
+        line-name = "enable-leds";
+        gpios = <10 GPIO_ACTIVE_HIGH>;
+        output-high;
+    };
+};
```

refer to e.g. https://patchwork.ozlabs.org/patch/1149065/ for further information.

This will prevent the user from changing this value, but in this case I do not consider this is relevant.

Reply:
@adrianschmutzler probably that gpio was used for power saving in the original firmware, in fact it enables/disables all leds except the power led. I always had leds enabled so it's not a problem for me, but I don't know if someone or some package would use this feature... Let me know what you think about it.

Reply:
Well, if you wanted to keep this adjustable in userspace, the way to go would be adding it to https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/base-files/etc/board.d/03_gpio_switches

In this case I'd _personally_ prefer the hog. The user should also still be able to configure the LEDs individually?

Reply:
Ok I added the hog. I don't think they would use the "power saving mode" anyway.

>The user should also still be able to configure the LEDs individually?

Probably not... these are standard leds